Dockerfile

```
@@ -0,0 +1,23 @@
+#
+# Build
+#
+FROM maven:3.9.6-amazoncorretto-17-al2023@sha256:459be099faa25a32c06cd45ed1ef2bc9dbbf8a5414da4e72349459a1bb4d6166 AS buildtime
+
+WORKDIR /build
+COPY . .
+
+RUN chmod +x ./gradlew
+RUN ./gradlew bootJar
+
+#
+# Docker RUNTIME
+#
+FROM amazoncorretto:17-alpine3.19@sha256:539a0a188ce5a2bed985aa311e9a26d473c6c3f37d08d4fc8b6cf6c18075b9ab AS runtime
```

the latest version I see having a different SHA:
https://hub.docker.com/layers/library/amazoncorretto/17-alpine3.19/images/sha256-070c3a37ea2465375014fbbdfbe0414d1e1a64339e1cc103d0059372bcc77647?context=explore
```suggestion
FROM amazoncorretto:17-alpine3.19@sha256:2122cb140fa94053abce343fb854d24f4c62ba3c1ac701882dce12980396b477 AS runtime
```